frontend/src/layouts/Layout.astro

```diff
@@ -0,0 +1,196 @@
+---
+export interface Props {
+  title: string;
+}
+
+const { title } = Astro.props;
+---
+
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width" />
+    <!-- <link rel="icon" type="image/svg+xml" href="/favicon.svg" /> -->
```

Comment:
let's remove this

Reply:
In case it isn't clear, I mean the comment

Reply:
I left it if we wanted to use an SVG for favicon instead?

Reply:
Good point, we should probably change it to SVG. I actually didn't know SVG favicons were supported by all major browsers

Reply:
Not the best https://caniuse.com/?search=favicon+svg...
https://css-tricks.com/how-to-favicon-in-2021/?